internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -35,6 +35,13 @@ import (
 
 var CosmosDbAccountResourceName = "azurerm_cosmosdb_account"
 
+var connStringPropertyMap = map[string]string{
+	"Primary SQL Connection String":             "primary_connection_string",
+	"Secondary SQL Connection String":           "secondary_connection_string",
+	"Primary Read-Only SQL Connection String":   "primary_readonly_connection_string",
+	"Secondary Read-Only SQL Connection String": "secondary_readonly_connection_string",
```

Comment:
since there's different interfaces to CosmosDB, presumably this should be:

```suggestion
	"Primary SQL Connection String":             "primary_sql_connection_string",
	"Secondary SQL Connection String":           "secondary_sql_connection_string",
	"Primary Read-Only SQL Connection String":   "primary_readonly_sql_connection_string",
	"Secondary Read-Only SQL Connection String": "secondary_readonly_sql_connection_string",
```

Reply:
Have updated